e2e/fixtures/pages/ExploreProfilesPage.ts

```diff
@@ -9,6 +10,17 @@ export class ExploreProfilesPage extends PyroscopePage {
     super(page, '/a/grafana-pyroscope-app/profiles-explorer', urlParams.toString());
   }
 
+  goto(explorationType?: string) {
```

Comment:
nit: could change the type here to `string | undefined` then you won't have to do `explorationType as string` below